omigami/feature_evaluator.py

```diff
@@ -69,14 +64,14 @@ def _get_feature_importances(self, estimator: Estimator):
             raise ValueError("The estimator provided has no feature importances")
 
     def _get_feature_ranks(
-        self, estimator: Estimator, features: Iterable[int]
+        self, estimator: Estimator, features: Union[List[int], NumpyArray]
     ) -> FeatureRanks:
         """Extract the feature rank from the input estimator. So far it can only handle
         estimators as scikit-learn ones, so either having the `feature_importances_` or
         the `coef_` attribute."""
         feature_importances = self._get_feature_importances(estimator)
         ranks = rankdata(-feature_importances)
-        return FeatureRanks(features=features, ranks=ranks, n_feats=self._n_features)
+        return FeatureRanks(features=features, ranks=ranks, n_feats=len(features))
```

Comment:
I think this could give you problems when performing averages among ranks with different `len(features)`